fe/fe-core/src/main/java/com/starrocks/catalog/TabletMeta.java

```diff
@@ -148,6 +150,10 @@ public boolean containsSchemaHash(int schemaHash) {
         }
     }
 
+    public boolean isLakeTablet() {
+        return isLakeTablet;
```

Comment:
the same as function name? may want to change a different variable name to avoid code read confusion.